text/0001-cdk-update.md

```diff
@@ -0,0 +1,293 @@
+---
+rfc pr: [#335](https://github.com/aws/aws-cdk-rfcs/pull/335)
+tracking issue: https://github.com/aws/aws-cdk-rfcs/issues/1
+---
+
+# CDK Update
+
+Rapid and automatic updates for code-only changes to CDK application stacks.
```

Comment:
The goal of this feature is to shorten the development iteration speed (also known as "inner loop"), so the scope of should be expanded. The feature is not just for code-only changes. We want CDK Watch to support things like API Gateway models, Step Functions, etc.